roles/lib_openshift/src/ansible/oadm_registry.py

```diff
@@ -0,0 +1,48 @@
+# pylint: skip-file
+# flake8: noqa
+
+def main():
+    '''
+    ansible oc module for registry
+    '''
+
+    module = AnsibleModule(
+        argument_spec=dict(
+            state=dict(default='present', type='str',
+                       choices=['present', 'absent']),
+            debug=dict(default=False, type='bool'),
+            namespace=dict(default='default', type='str'),
+            name=dict(default=None, required=True, type='str'),
+
+            kubeconfig=dict(default='/etc/origin/master/admin.kubeconfig', type='str'),
+            credentials=dict(default='/etc/origin/master/openshift-registry.kubeconfig', type='str'),
+            images=dict(default=None, type='str'),
+            latest_images=dict(default=False, type='bool'),
```

Comment:
@detiber, We can remove but I would ask if customers find this option useful.  I only mapped the oc adm registry api and this was one of the options.  If we find that we aren't using it, I'm fine with removing it.